arb/test/t-nth_root.c

```diff
@@ -20,11 +23,16 @@ int main() {
             slong m, d, j;
 
             d = 3 + n_randint(state, 20);
-            m = n_randint(state, 12);
+            m = 2 + n_randint(state, 12);
             u = _acb_vec_init(d);
             arb_init(x);
             acb_init(y1);
             acb_init(y2);
+            acb_init(y3);
+            acb_init(z);
+            acb_set_ui(z,2);
```

Comment:
so you can use _acb_vec_unit_roots with order 2m here, z being a _acb_vec_init(2*m)